terraform/transform_output.go

```diff
@@ -37,65 +43,60 @@ func (t *OutputTransformer) transform(g *Graph, c *configs.Config) error {
 		}
 	}
 
-	// Add plannable outputs to the graph, which will be dynamically expanded
+	// Add outputs to the graph, which will be dynamically expanded
 	// into NodeApplyableOutputs to reflect possible expansion
 	// through the presence of "count" or "for_each" on the modules.
-	for _, o := range c.Module.Outputs {
-		node := &nodeExpandOutput{
-			Addr:   addrs.OutputValue{Name: o.Name},
-			Module: c.Path,
-			Config: o,
-		}
-		log.Printf("[TRACE] OutputTransformer: adding %s as %T", o.Name, node)
-		g.Add(node)
-	}
-
-	return nil
-}
-
-// destroyRootOutputTransformer is a GraphTransformer that adds nodes to delete
-// outputs during destroy. We need to do this to ensure that no stale outputs
-// are ever left in the state.
-type destroyRootOutputTransformer struct {
-	Destroy bool
-}
 
-func (t *destroyRootOutputTransformer) Transform(g *Graph) error {
-	// Only clean root outputs on a full destroy
-	if !t.Destroy {
-		return nil
+	// if this is a root output, we add the apply or destroy node directly, as
```

Comment:
This comment doesn't match the following code on my read (it's related to changes, not a root check)

Reply:
you're right, that got stale. I'll move it down closer to the appropriate context.